dataframely/collection.py

```diff
@@ -620,84 +625,230 @@ def serialize(cls) -> str:
+                  found in the directory, this method automatically runs
+                  :meth:`validate` with ``cast=True``. However, it prints a warning
+                  that the read introduces additional overhead.
+                - If ``True``, the method behaves similarly to ``"auto"``. However, the
+                  user acknowledges that the read might run validation, and no warning
+                  will be emitted.
+                - If ``False``, validation is never run automatically and an error is
+                  raised if the parquet file does not store schema information or the
+                  stored schema mismatches this schema.
+
+            kwargs: Additional keyword arguments passed directly to
+                :meth:`polars.scan_parquet` for all members.
 
         Returns:
             The initialized collection.
 
         Raises:
+            ValidationRequiredError: If no collection schema can be read from the
+                directory and ``validate`` is set to ``False``.
             ValueError: If the provided directory does not contain parquet files for
                 all required members.
 
         Note:
-            If you want to eagerly read all Parquet files, consider calling
-            :meth:`collect_all` on the returned collection.
+            Due to current limitations in dataframely, this method actually reads the
+            parquet file into memory if ``validate`` is ``"auto"`` or ``True`` and
+            validation is required.
 
         Attention:
-            This method does **not** validate the contents of the Parquet file. Consider
-            using :meth:`read_parquet` if you want to validate the collection.
+            Be aware that this method suffers from the same limitations as
+            :meth:`serialize`.
         """
         data = {
-            key: pl.scan_parquet(directory / f"{key}.parquet")
+            key: pl.scan_parquet(directory / f"{key}.parquet", **kwargs)
             for key in cls.members()
             if (directory / f"{key}.parquet").exists()
         }
-        cls._validate_input_keys(data)
-        return cls._init(data)
+        if not cls._requires_validation_for_reading_parquets(directory, validate):
+            cls._validate_input_keys(data)
+            return cls._init(data)
+        return cls.validate(data, cast=True)
+
+    @classmethod
+    def _requires_validation_for_reading_parquets(
+        cls,
+        directory: Path,
+        validate: bool | Literal["auto"] = "auto",
+    ) -> bool:
+        # First, we check whether the path provides the serialization of the collection.
+        # If it does, we check whether it matches this collection. If it does, we assume
+        # that the data adheres to the collection and we do not need to run validation.
+        if (json_serialization := directory / "schema.json").exists():
+            with json_serialization.open("r") as f:
+                metadata = f.read()
```

Comment:
```suggestion            
            metadata = json_serialization.read_text()
```
no biggie, but why not?